src/pages/ps_reference/changelog/index.md

```diff
@@ -28,6 +28,77 @@ contributors:
+  - Event `uxpvideoplay` in [play()](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLVideoElement.md#play) is deprecated. Use `ended` instead.
+  - Event `uxpvideocomplete` in [play()](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLVideoElement.md#play) is deprecated. Use `ended` instead.
+  - Event `uxpvideopause` in [pause()](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLVideoElement.md#pause) is deprecated. Use `pause` instead.
+
+#### New
+- New components supported in Spectrum Web Components (SWC). More details [here](https://github.com/adobe/swc-uxp-wrappers/tree/main?tab=readme-ov-file#spectrum-web-components-uxp-wrappers)
+  - sp-asset
+  - sp-meter
+  - sp-number-field
+  - sp-overlay
+  - sp-quick-actions
+  - sp-search
+  - sp-swatch
+- New command line templates to get started with SWC easily. Find details in this [section](../../uxp-api/reference-spectrum/swc/index.md#command-line-templates).
+- Support for local HTML files in [WebView](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLWebViewElement.md)
+- New CSS capabilities
+  - scaleX() & scaleY()
+  - transform-origin
+  - translate property
+  - `Box-shadow` property now supports `transform rotate`.
+- New feature flag `CSSNextSupport` in the manifest.json to gate the new and future CSS capability.
+
+  | Manifest setting | Description  |
+  | ---------------- | ------------ |
+  | `featureFlags: {"CSSNextSupport": true}` | Enables all the new CSS features. |
+  | `featureFlags: {"CSSNextSupport": ["boxShadow", "transformFunctions", "transformProperties" ]}` | Enables only the named properties in the list. |
+  
+  Note that `"enableSWCSupport": true` will enable `CSSNextSupport` by default, even if it is not done explicitly in plugin manifest otherwise.
+- Video enhancements: Support added for [Seek Event](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLVideoElement.md)
+- Support for [Navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language) API
+- Support for new HTMLElement APIs - [append](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLElement.md#appendnodes), [prepend](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLElement.md#prependnodes) and [replaceChildren](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLElement.md#replacechildrennodes)
+- [Beta] Support for form reset in [HTMLFormElement](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form)
+
+#### Updated
+- Upgraded Spectrum Web Components to v0.37.0. [More detail](https://github.com/adobe/swc-uxp-wrappers/tree/main?tab=readme-ov-file#spectrum-web-components-uxp-wrappers)
+- [UXP Clipboard.writeText()](../../uxp-api/reference-js/Global%20Members/Data%20Transfers/Clipboard.md#writetexttext) now accepts strings as input to meet standards
+
```

Comment:
This should move up under the heading:

# Photoshop API Changelog

Reply:
Fixed